frontend/src/main/components/Leaderboard/LeaderboardTable.js

```diff
@@ -24,10 +24,16 @@ export default function LeaderboardTable({ leaderboardUsers , currentUser }) {
             accessor: (row, _rowIndex) => {
                 return USD.format(row.totalWealth);
             },
+            // sortType: 
```

Comment:
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

Reply:
Got it, I will delete the commented-out code for this PR.